tools/verify_docker_gui.sh

```
@@ -0,0 +1,208 @@
+Host contract:
+  - Linux host with an active X11 DISPLAY
+  - /tmp/.X11-unix available on the host
+  - Optional XAUTHORITY cookie file, exported via FRAME_COMPARE_XAUTHORITY_PATH
+  - Container runs as the host UID/GID so narrow local-user X11 permissions apply
+
+Manual UI launch remains separate from this proof.
+EOF
+}
+
+readonly DEFAULT_SERVICE="frame-compare-test"
+readonly REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
```

Shellcheck (0.11.0)

[warning] REPO_ROOT appears unused. Verify use (or export if used externally).

(SC2034)